### PR TITLE
Zip workflow: Upload to quay

### PR DESCRIPTION
**Pull request by MartinBasti**

### CODE REVIEW

File: omps/quay.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+        :param organizations: dictionary with configuration
+        :raises jsonschema.ValidationError: when configuration doesn't meet
+            expectations
+        """
+        jsonschema.validate(organizations, cls.SCHEMA_ORGANIZATIONS)
+
+    def __init__(self):
+        self._organizations = {}
+        self._quay_url = "https://quay.io"
```
Comment (athos-ribeiro) [inline omps/quay.py]: should this URL be configurable somewhere?

File: Dockerfile [hunk 2]
```diff
@@ -13,6 +13,7 @@ RUN dnf -y install \
     python3-flask \
     python3-jsonschema \
     python3-requests \
+    https://kojipkgs.fedoraproject.org//packages/python-operator-courier/1.0.0/1.fc29/noarch/python3-operator-courier-1.0.0-1.fc29.noarch.rpm \
```
Comment (athos-ribeiro) [inline Dockerfile]: This should change as soon as https://bodhi.fedoraproject.org/updates/FEDORA-2019-72022ab81e gets pushed to stable (or updates-testing could be enabled here).

File: omps/quay.py (hunk 1 above)
Comment (MartinBasti) [reply omps/quay.py]: I originally wanted to make it configurable, but `operator-courier` module has hardcoded that URL

File: Dockerfile (hunk 2 above)
Comment (MartinBasti) [reply Dockerfile]: I prefer to not enable updates-testing, I will change it to `python3-operator-courier` once it gets into stable

**Pull request merged**